farming/logics/traits/data.rs

```diff
@@ -0,0 +1,54 @@
+use ink_prelude::vec::Vec;
+use ink_storage::{
+    traits::*,
+    Mapping,
+};
+use openbrush::traits::AccountId;
+use scale::{
+    Decode,
+    Encode,
+};
+
+pub const STORAGE_KEY: u32 = openbrush::storage_unique_key!(Data);
+
+#[derive(Encode, Decode, SpreadLayout, PackedLayout, SpreadAllocate, Default)]
+#[cfg_attr(
+    feature = "std",
+    derive(Debug, PartialEq, Eq, scale_info::TypeInfo, StorageLayout)
```

Comment:
Why do we derive Debug, etc. only when std is loaded? `scale_info::TypeInfo` and `StorageLayout` are the ones that need std library (which is for meta data), but not for the others.

Reply:
Good point. This is due to copy paste since months